bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -39,7 +39,10 @@ where:
 - See [Step\(s\)](#steps) for an explanation of `<step>`, `<consolidationfunction>`, `<xff>`, `<steps>`, and `<rows>`.
 - `<list of items for this dsname>` is explained in
 
-Round-robin databases (RRDs) have fixed-length so-called "archives" for storing values. One RRD can have (in general) several datasources and each datasource can have several archives. openHAB only supports one datasource per RRD, which is named DATASOURCE_STATE.
+Round-robin databases (RRDs) have fixed-length so-called "archives" for storing values. 
+One RRD can have (in general) several datasources and each datasource can have several archives. 
+openHAB only supports one datasource per RRD (i.e. per stored item), which is named DATASOURCE_STATE.
+The setup of multiple configurations (with differing .def ,.archives and .items settings) can be used if each item is member of exact one setup only (see example below).
```

Comment:
Let's replace <s>The setup of multiple</s> with just "Multiple"

Also:
".def, .archives"

and

"if each item is a member of exactly one setup"